src/assets/wallets.md

```diff
@@ -97,6 +97,7 @@ curl -X GET "https://service.centrapay.com/api/wallets/WRhAxxWpTKb5U7pXyxQjjY/tr
 
 | Field            | Description                                                               |
 |:-----------------|:--------------------------------------------------------------------------|
+| activityNumber   | Incrementing value indicating the number of transactions on the wallet.   |
```

Comment:
Can we word this as "unique sequential wallet transaction number"?

Reply:
Sure, I've made this change